en/docs/Compliance/general-Data-Protection-Regulation.md

```diff
@@ -35,34 +16,30 @@ Server. This section outlines the major GDPR related features that WSO2
 Identity Server supports, and also provides necessary references to
 specific concepts within the section.
 
-This section also includes an
-[FAQ](https://docs.wso2.com/display/IS580/GDPR+FAQ) page that can give
-you more specific and direct answers to any question you may have on
-GDPR as well as on GDPR compliance that WSO2 offers.
+!!! Info
 
-Do you want to learn more about GDPR?
+	  Do you want to learn more about GDPR?
 
-If you are new to GDPR, we recommend that you take a look at our
-tutorial series on ***Creating a Winning GDPR Strategy.***
+	  If you are new to GDPR, we recommend that you take a look at our
+	  tutorial series on *Creating a Winning GDPR Strategy*.
 
--   Part 1 - [Introduction to
-    GDPR](https://wso2.com/library/article/2017/12/introduction-to-gdpr/)
+	  -   Part 1 - [Introduction to
+    	  GDPR](https://wso2.com/library/article/2017/12/introduction-to-gdpr/)
 
--   Part 2 - [7 Steps for GDPR
-    Compliance](https://wso2.com/library/article/2017/12/7-steps-for-gdpr-compliance/)
+	  -   Part 2 - [7 Steps for GDPR
+  	  Compliance](https://wso2.com/library/article/2017/12/7-steps-for-gdpr-compliance/)
 
--   Part 3 - [Identity and Access Management to the
-    Rescue](https://wso2.com/library/article/2018/2/identity-and-access-management-to-the-rescue/)
+	  -   Part 3 - [Identity and Access Management to the
+    	  Rescue](https://wso2.com/library/article/2018/2/identity-and-access-management-to-the-rescue/)
 
--   Part 4 - [GDPR Compliant Consent
-    Design](https://wso2.com/library/articles/2018/03/creating-a-winning-gdpr-strategypart-4-gdpr-compliant-consent-design/)
+	  -   Part 4 - [GDPR Compliant Consent
+    	  Design](https://wso2.com/library/articles/2018/03/creating-a-winning-gdpr-strategypart-4-gdpr-compliant-consent-design/)
 
-For more resources on GDPR, see the white papers, case studies, solution
-briefs, webinars, and talks published on our [WSO2 GDPR
-homepage](https://wso2.com/solutions/regulatory-compliance/gdpr/) . You
-can also find the original GDPR legal text
-[here](http://eur-lex.europa.eu/legal-content/en/TXT/?uri=CELEX%3A32016R0679)
-.
+	  For more resources on GDPR, see the white papers, case studies, solution
+	  briefs, webinars, and talks published on our [WSO2 GDPR
+	  homepage](https://wso2.com/solutions/regulatory-compliance/gdpr/) . You
```

Comment:
An extra space exists just before the period.

Reply:
Fixed with commit 895dc3b5173232dd1a2cd561fb9d5b264bb096bb